platform_versioned_docs/version-24.2/troubleshooting_and_faqs/nextflow.mdx

```diff
@@ -225,4 +225,4 @@ If you rely on silent Spot retries (the previous default behavior), you may now
 - **AWS**: Generic failure with `exit code 1`. You may see messages indicating the host machine was terminated.
 - **Google**: Spot reclamation typically produces a specific code, but is now surfaced as a recognizable task failure in Nextflow logs.
 
-Since the default for Spot retries is now zero, you must actively enable a retry strategy if you want Nextflow to handle reclaimed Spot Instances automatically. For more information, see the [Spot Instance failures and retries](https://www.nextflow.io/docs/latest/updating-spot-retries.html) guide. 
+Since the default for Spot retries is now zero, you must actively enable a retry strategy if you want Nextflow to handle reclaimed Spot instances automatically. For more information, see the [Spot instance failures and retries](https://www.nextflow.io/docs/latest/updating-spot-retries.html) guide. 
```

Comment:
```suggestion
Since the default for Spot retries is now zero, you must actively enable a retry strategy if you want Nextflow to handle reclaimed Spot instances automatically. For more information, see the [Spot Instance failures and retries](https://www.nextflow.io/docs/latest/updating-spot-retries.html) guide. 
```